src/display/base_factory.js

```diff
@@ -143,14 +143,18 @@ class BaseSVGFactory {
     }
   }
 
-  create(width, height) {
+  create(width, height, skipDims = false) {
```

Comment:
I'd prefer we call this `skipDimensions` for better readability here and in the callsites.